app/models/katello/deb.rb

```diff
@@ -95,6 +96,13 @@ def self.applicable_to_hosts(hosts)
         where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).distinct
     end
 
+    # Return deb packages that are not installed on a host, but could be installed
+    # the word 'installable' has a different meaning here than elsewhere
+    def self.deb_installable_for_host(host)
```

Comment:
Thoughts on calling it this instead? It's more in line with "yum installable" which is what we called it on the rpm side.

```suggestion
    def self.apt_installable_for_host(host)
```